### PR TITLE
adds linux and mac desktop apps; alerts users when apps not ready yet

### PR DESCRIPTION
**Pull request by owocki**
##### Description

adds linux and mac desktop apps; alerts users when apps not ready yet

##### Refers/Fixes

slack convo

##### Testing

tested locally

### CODE REVIEW

File: app/assets/v2/js/pages/chat.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+$(document).ready(function() {
+  $("a").click(function(e){
```
Comment (octavioamu) [inline app/assets/v2/js/pages/chat.js]: this is goiing to block any link on that page, why we just don't comment out the download section?

Comment (owocki) [reply app/assets/v2/js/pages/chat.js]: it wont, only ones that have an `#` as their href will have `e.preventDefault()`

Comment (owocki) [reply app/assets/v2/js/pages/chat.js]: see the condition on line 5

Comment (androolloyd) [reply app/assets/v2/js/pages/chat.js]: I feel this selector is a bit too global, giving that there are other "#" links

Screen Shot 2020-01-10 at 1 40 42 PM


There would be more than just the links we want to say coming soon caught under this, selector.

What about doing something like this. 

`$(".my-3 a[href='#']").on('click', (e) => {alert("Coming Soon!"); e.preventDefault(); return;})`

Will only target the anchors inside the my-3 divs section on this page.

Comment (owocki) [reply app/assets/v2/js/pages/chat.js]: 325e042ff311224688c6ff4adbf86a2e6a72d290

**Comment by octavioamu** [comment]
Approved, do you want to merge this one to master? not stable?

**Comment by owocki** [comment]
ill cherrypick to stable!

**Pull request merged**